src/main/java/dukeApp/files/Event.java

```diff
@@ -0,0 +1,70 @@
+package dukeApp.files;
+
+public class Event extends Task {
+    protected String word;
+    protected String date;
+
+    public Event(String s) {
+        super(s);
+        description = s.split("/")[0];
+        date = s.split("/")[1];
+        word = date.substring(0, date.indexOf(" "));
+        date = date.substring(date.indexOf(" ") + 1);
+    }
+
+    /**
+     * Returns the type of task
+     *
+     * @return the letter indicating the task type
+     */
+    public String getType() {
+        return "E";
+    }
+
+    /**
+     * Returns the task date description
+     *
+     * @return task date description
+     */
+    public String getTask() {
```

Comment:
Instead of returning description through getTask(), you can override toString method, then when you print the instance e.g. System.out.println(task), it'll automatically print the description without needing to use any methods.